nxviz/plots.py

```diff
@@ -459,15 +473,68 @@ def compute_node_label_positions(self):
             else:  # left side
                 rot = theta_deg - 180
 
-            xs.append(x)
-            ys.append(y)
-            has.append(ha)
-            vas.append(va)
-            rotations.append(rot)
+            # Store values
+            self._store_node_label_meta(x, y, tx, ty, rot)
+
+    @staticmethod
+    def nz_sign(xy):
+        return -1 if xy < 0 else 1
+
+    def _init_node_label_meta(self):
```

Comment:
I like the extra checks that this method provides. Have you ran the tests and verified that this change doesn't break anything, and ran a few of the examples to visually check as well?